pycryptobot.py

```diff
@@ -988,13 +988,16 @@ def main(init=True):
 
         print(repr(e))
 
-        # Wait 1 second and try to relaunch application
-        time.sleep(1)
-        print('Restarting application after exception...')
+        if app.autoRestart():
```

Comment:
We should not be doing this as scheduling is handled 1 level higher and is sitting in pycryptobot.py.
We should put a try catch on scheduler and try invoke executeJob again 30 sec later.

Reply:
I didn't see a response to this. How do you propose it is done?

Reply:
Sorry I don't have time now to change it, I'll try later to make the try/except around the s.run()

Reply:
@letsautom8, what did you mean by your suggestion. Can you advise the correct way to do it?

Reply:
Making the auto restart option optional would really help. What is happening with this PR?

Reply:
I did change to try/except around the scheduler.
Sorry for the delay I was not available.

Reply:
It's okay, thanks very much for sorting this out.